lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -477,8 +478,52 @@ def __get_template_with_version(self):
 
         return template
 
+    def __get_storage_domain(self, disk_id):
+        storage_domain = None
+
+        disks = self._module.params.get('disks', list())
+
+        for disk in disks:
+            if disk_id == disk.get('id'):
```

Comment:
Can we somehow implement behavior of specify storage_domain for all disks? If I understand the code correctly, it not currently.